src/zenml/orchestrators/step_launcher.py

```diff
@@ -158,7 +161,22 @@ def launch(self) -> None:
         logger.info(f"Step `{self._step_name}` has started.")
 
         pipeline_run, run_was_created = self._create_or_reuse_run()
+
+        # Set up logging
+        logs_uri = step_logging_utils.prepare_logs_uri(
+            self._stack.artifact_store,
+            pipeline_run.id,
+            self._step.config.name,
+        )
+
+        zenml_handler = step_logging_utils.get_step_logging_handler(
+            self._stack.artifact_store, logs_uri
+        )
+
         try:
+            root_logger = logging.getLogger()
```

Comment:
This is what we do in other files:

```python
from zenml.logger import get_logger

logger = get_logger(__name__)
```

Reply:
@fa9r That logger is for us only, the root logger is what I imagine most people will use. It is the thing you use when you do :

```
import logging

logging.info("...")
```

Reply:
Ah yeah makes sense. Not sure about this myself, but could there be any issues if we log step logs with this instead of the ZenML logger now?

Reply:
Not sure maybe @schustmi has ideas but i also unregister the handler at the end of the step to avoid this. There is a gap however after setting up this handler and the step actually running where we could run some code that could leak youre right

Reply:
"Leak" as in "ZenML logs will show up in the dashboard instead of just the logs happening inside the step function? I'm not sure if this is an issue or even what we want as it might help users debug in case it crashes there?

Reply:
Yep, however I would say those are also expected somehow

Reply:
ok